### ISSUE TITLE
Migrations are ignored

### ISSUE THREAD

**Issue by Vandaahl** [Sylius/RefundPlugin#237]
I have installed a fresh copy of Sylius Standard Edition version 1.8.7. Then I install the Refund plugin according to the instructions:

`composer require sylius/refund-plugin`

I accept the recipe and run the migration:

`bin/console doctrine:migrations:migrate`

The output is as follows:

> [OK] Already at the latest version ("Sylius\Bundle\AdminApiBundle\Migrations\Version20170711151342")

My src/Migrations folder is empty. No database changes are made. When I copy the migration files from vendor/sylius/refund-plugin/migrations/* to src/Migrations and run the migration command again, nothing changes because I get the same output as above.

I do not have alot of Symfony or Sylius experience yet so I'm not sure where the problem lies. I think it's a bug though.

**Comment by mxkh** [comment]
I can confirm this issue. Everything was done following the official documentation

**Comment by Zales0123** [comment]
Hi! I believe there is a problem in the documentation. When you run just `composer require sylius/refund-plugin` the `0.10.1` version is installed by default. To install the latest, RC-5 version, you need to run `composer require sylius/refund-plugin:v1.0.0-RC.5`. It would not be a problem after the stable release :)

Are you willing to open a PR with such a change? Thanks! 🖖

**Comment by Vandaahl** [comment]
Thanks Zales0123, I was able to install the plugin correctly. I'm not yet familiar with doing pull requests, so I'll look into it.

Having installed the plugin, I'm trying it out and run into an issue with partial refunds. When I open an order, press "Refunds" and submit a partial refund, it looks like it's working but it does display an "**Unexpected error occurred**" message on top. I tried with both default payment methods (bank transfer and cash on delivery).
